src/layout/macros.rs

```diff
@@ -137,7 +137,7 @@ macro_rules! CType {(
                         $crate::__output_docs__!(out, "", $($doc_meta)*);
                     )?
                     $crate::__output_docs__!(out, "", $(#[$($meta)*])*);
-                    $crate::core::writeln!(out, "typedef struct {{\n")?;
+                    $crate::core::writeln!(out, "typedef struct {}_t {{\n", me)?;
```

Comment:
Would the following work? It's a bit more idiomatic not to bundle the `_t` when using the long `struct …` name 🙂

```suggestion
                    $crate::core::writeln!(out, "typedef struct {} {{\n", me)?;
```

Reply:
This works! Thank you!!